### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T304338)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# WPF Dock Layout Manager - Automatically Merge Ribbon Controls
+
+The following control support automatic bar merging:
```
Comment (albertov05) [inline Readme.md]:
```suggestion
The following controls support automatic bar merging:
```

File: Readme.md [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
+
+* [DXTabControl](https://docs.devexpress.com/WPF/7975/controls-and-libraries/layout-management/tab-control/fundamentals/dxtabcontrol)
+* [Dock Layout Manager](https://docs.devexpress.com/WPF/6191/controls-and-libraries/layout-management/dock-windows)
+  
+In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contains ribbon controls.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contain ribbon controls.
```

File: Readme.md [hunk 3]
```diff
@@ -1,28 +1,40 @@
...
+
+The example illustrates pages, page groups and items merging from child ribbons to the main ribbon control.
```
Comment (albertov05) [inline Readme.md]: Do you mean the following?

`The example illustrates how pages, page groups, and items are merged from child ribbons to the main ribbon control.`

Comment (friedcucumber) [reply Readme.md]: The example merges a pages, page groups, and items from child ribbons to the main ribbon control.

Comment (friedcucumber) [reply Readme.md]:
```suggestion
The example merges a pages, page groups, and items from child ribbons to the main ribbon control.
```

File: Readme.md [hunk 4]
```diff
@@ -1,28 +1,40 @@
...
+# WPF Dock Layout Manager - Automatically Merge Ribbon Controls
+
+The following controls support automatic bar merging:
+
+* [DXTabControl](https://docs.devexpress.com/WPF/7975/controls-and-libraries/layout-management/tab-control/fundamentals/dxtabcontrol)
+* [Dock Layout Manager](https://docs.devexpress.com/WPF/6191/controls-and-libraries/layout-management/dock-windows)
+  
+In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contain ribbon controls.
+
+The example merges a pages, page groups, and items from child ribbons to the main ribbon control.
```
Comment (albertov05) [inline Readme.md:14]: Can you update this line as follows?

`The example merges pages, page groups, and items from child ribbons to the main ribbon control.`

**Pull request merged**